recipes-bsp/bootfiles/rpi-config.bb

```diff
@@ -36,6 +36,13 @@ GPIO_SHUTDOWN_PIN ??= ""
 
 inherit deploy nopackages
 
+RASPBERRYPI_CAMERA ??= "auto"
+RASPBERRYPI_CAMERA_V1 = "ov5647"
+RASPBERRYPI_CAMERA_V2 ?= "imx219"
+RASPBERRYPI_CAMERA_HQ = "imx477"
```

Comment:
What uses some of these variables?

Reply:
These variables describe the common values and are optionally used by the user to assign RASPBERRYPI_CAMERA if they wish not to explicit the actual model name.

Reply:
Can you give me an example?

Reply:
`RASPBERRYPI_CAMERA = "${RASPBERRYPI_CAMERA_V2}"`, it's optional, but convenient in order to avoid case sensitive errors or other typos. I'll update the doc to reflect that as well.

Reply:
I propose keeping this simple with only RASPBERRYPI_CAMERA and giving documentation examples of its values. Having indirect definitions and keeping them updated with new variables is a situation that might cause more confusion.